app/sprinkles/core/src/Exceptions/VersionCompareException.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * UserFrosting (http://www.userfrosting.com)
+ *
+ * @link      https://github.com/userfrosting/UserFrosting
+ * @copyright Copyright (c) 2019 Alexander Weissman
+ * @license   https://github.com/userfrosting/UserFrosting/blob/master/LICENSE.md (MIT License)
+ */
+
+namespace UserFrosting\Sprinkle\Core\Exceptions;
+
+use LogicException;
+
+/**
+ * Version Compare Exception. Used when a class name is dynamically invoked, but the class does not exist.
```

Comment:
This description makes no sense.

Reply:
Forgot to change after copy and paste...